packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -139,7 +139,8 @@ def wait_for_container_propogation():
                 cid_registry = []
                 for dp in container_response.json()['datapoints']:
                     assert 'tags' in dp, 'got {}'.format(dp)
-                    assert len(dp['tags']) == 5, 'got {}'.format(
+                    # blkio stats have 'device' tags as well
+                    assert len(dp['tags']) >= 5, 'got {}'.format(
```

Comment:
If we've added blkio tags/metrics, do we want to test that `len(dp['tags']) > 5` instead?

Reply:
Not all the container stats are blkio stats - the existing (mem, cpu, gpu, network I/O) stats don't have blkio source device tags, so they still have five.

Reply:
Aha, I get it now -- thanks.